config/template_values.yml

```diff
@@ -4,42 +4,42 @@
 # the db (famous last words).  Rule of thumb: If you need to know the value
 # behind the formatted text, don't put it here.
 
-install_day: "Thursday, July 4, 2024"
-skill_attend_day: "Friday, June 30, 2024"
+install_day: "Thursday, July 4, 2025"
```

Comment:
july 4th is friday.  all the stupid days of the week change :(

Reply:
good catch! Checked all the remaining dates as well